apps/web/src/forms/QuestionSteps.tsx

```diff
@@ -185,35 +171,43 @@ export const QuestionSteps: Step[] = [
       <SymptomQuestion
         answerOptions={Object.entries(AgeRanges).map((key, val) => {
           const index = val + 1;
-          return { key: `HWQ7-${index}`, value: key[1] };
+          return { key: `HWQ6-${index}`, value: key[1] };
         })}
         question={{
-          title: 'HWQ7',
+          title: 'HWQ6',
         }}
         name='healthWork.age'
       />
     ),
     key: 'What is your age?',
   },
-];
-
-// return the final step of the survey, dependent on prior questions
-export const LastStep = (isMultiple: boolean): Step => {
-  const questionTitle = isMultiple ? 'HWQ8-M' : 'HWQ8-S';
-  return {
+  {
     type: QuestionType.HEALTH_WORK,
     component: (
       <SymptomQuestion
         answerOptions={YES_NO_OPTIONS}
         question={{
-          title: questionTitle,
-          description: 'HWQ8-desc',
+          title: 'HWQ7-M',
+          description: 'HWQ7-desc',
         }}
         name='healthWork.chronicConditions'
       />
     ),
     key: 'Do you have chronic conditions?',
-  };
-};
-
-export const numberOfQuestions = QuestionSteps.length + 1;
+  },
+  {
+    type: QuestionType.HEALTH_WORK,
+    component: (
+      <SymptomQuestion
+        answerOptions={YES_NO_OPTIONS}
+        question={{
+          title: 'HWQ7-S',
+          description: 'HWQ7-desc',
+        }}
+        name='healthWork.chronicConditions'
+      />
+    ),
+    key: 'Do you have chronic conditions?',
```

Comment:
Just double checking there are two questions that should have the same key (this and previous one)

Reply:
The difference between HWQ7-S and HWQ7-M is that in two cases (7 and 8) where the wording of the question changes (Do you have chronic conditions vs do you have 3 or more chronic conditions).